Examples/Objective-C/ViewController.m

```diff
@@ -3,32 +3,29 @@
 @import AVFoundation;
 @import MapboxNavigation;
 @import MapboxDirections;
+@import MapboxNavigationUI;
 @import Mapbox;
 
 @interface ViewController () <AVSpeechSynthesizerDelegate>
 @property (nonatomic, weak) IBOutlet MGLMapView *mapView;
-@property (weak, nonatomic) IBOutlet UIView *instructionsView;
-@property (weak, nonatomic) IBOutlet UILabel *instructionsLabel;
+@property (weak, nonatomic) IBOutlet UIButton *toggleNavigationButton;
+@property (weak, nonatomic) IBOutlet UILabel *howToBeginLabel;
 @property (nonatomic, assign) CLLocationCoordinate2D destination;
 @property (nonatomic) MBDirections *directions;
+@property (nonatomic) MBRoute *route;
 @property (nonatomic) MBRouteController *navigation;
 @property (nonatomic) NSLengthFormatter *lengthFormatter;
 @property (nonatomic) AVSpeechSynthesizer *speechSynth;
 @end
 
 @implementation ViewController
 
-static NSString *MBXTempProfileIdentifierAutomobileAvoidingTraffic = @"mapbox/driving-traffic";
-static NSString *MapboxAccessToken = @"<#Your Mapbox access token#>";
-
 - (void)viewDidLoad {
     [super viewDidLoad];
     // Do any additional setup after loading the view, typically from a nib.
+
+    [self.mapView setStyleURL:[MGLStyle streetsStyleURLWithVersion:9]];
```

Comment:
This is inconsistent with the storyboard in the Swift example project, which uses Streets v10.